pages/[id].jsx

```diff
@@ -31,7 +31,8 @@ export default function DetailPage({ fetchedTeamMember }) {
 export async function getServerSideProps({ params, req }) {
   const { id } = params;
   const { host } = req.headers;
-  const baseUrl = `http://${host}/api/fellowship/`;
+  const scheme = process.env.NODE_ENV === 'development' ? 'http://' : 'https://';
+  const baseUrl = `${scheme}${host}/api/fellowship/`;
```

Comment:
This isn't exactly the fix the issue described, but exporting the environment through `config/index.js` was not working as expected and NextJS ships with support for environment variables so this seemed like a good way to go to me. As always if there is something I am missing I am happy to make the changes.

Reply:
Okay, this works for now. I would like to see a `.env.example`, though that we can add to as we need environment variables.

```
# set to 'development' for local, anything else will be considered production
NODE_ENV=development
```

Reply:
Sounds good! I pushed the addition